### PR TITLE
Adding Rancher instructions to README.md

### PR DESCRIPTION
**Pull request by abonillabeeche**
Added instructions to Install S3GW via Rancher

Signed-off-by: Alejandro Bonilla <abonilla@suse.com>

### CODE REVIEW

**Comment by giubacc** [comment]
Thanks @abonillabeeche for contributing!
I think adding documentation specific for Rancher is a great idea.
I don't think btw, that this is the best place to add this kind of documentation; I'm more of the opinion to add a specific section/file about Rancher under our documentation at https://github.com/aquarist-labs/s3gw/tree/main/docs.
What do you think @jhmarina @jecluis ?
We also encourage opening an issue here https://github.com/aquarist-labs/s3gw for tracking purpose.

**Comment by jhmarina** [comment]
> ... What do you think @jhmarina @jecluis ? We also encourage opening an issue here https://github.com/aquarist-labs/s3gw for tracking purpose.

-- 

Yes, the documentation should be added here: https://github.com/aquarist-labs/s3gw/tree/main/docs

The files in that folder are then used to generate this page: https://s3gw-docs.readthedocs.io/en/latest/

There's an issue created for moving this [documentation to the main `docs ` folder](https://github.com/aquarist-labs/s3gw/issues/81) we can use the issue to add these instructions too.

I'll close this PR and add it to the issue.

**Comment by abonillabeeche** [comment]
When the chart is added into Rancher, it reads the README and shows the chart specific instructions, which allows the user to follow/have instructions at hand as they initially try and deploy the chart. If I add it where I did, the user will see it as they are in the process of installing via Rancher.

**Comment by giubacc** [comment]
Personally, I would agree to have a minimum of documentation at hand.
But the team, at a certain point, decided to aggregate all the documentation and centralize it elsewhere.
We can discuss if a mixed approach could be better; but for the time being, we have that rule.

**Comment by jhmarina** [comment]
I don’t have any issue with having minimal documentation available in the readme file. 

What we agreed on is having basic “Getting started” information on it and then all the remaining information in the documentation. 

Perhaps the info added in the PR should stay here

**Comment by jecluis** [comment]
I think it makes sense to have some information on each repository, because otherwise potential users can be confused. Having even minimal information, or information that is relevant to have, and then having a link to the full documentation seems the way to go IMO.

So I don't see a problem with merging this PR.

**Comment by jhmarina** [comment]
I reopened the PR. @abonillabeeche you’ll need to sign it off though. DCO is complaining

File: charts/s3gw/README.md [hunk 1]
```diff
@@ -12,6 +12,17 @@ Then change into the repository and install using Helm:
     cd s3gw-charts
     helm install $RELEASE_NAME charts/s3gw --namespace $S3GW_NAMESPACE --create-namespace -f /path/to/your/custom/values.yaml
 
+### Rancher ###
+
+Installing s3gw via the Rancher App Catalog is made easy, the steps are as follow
+
+    -> Cluster -> Projects/Namespaces - create the s3gw namespace
+    -> Storage -> PersistentVolumeClaim -> Create -> choose the s3gw namespace -> provide a size and name it s3gw-pvc
+    -> Apps -> Repositories -> Create S3GW using the s3gw-charts Git URL https://github.com/aquarist-labs/s3gw-charts and the main branch
```
Comment (giubacc) [inline charts/s3gw/README.md:21]: for consistency, I would rather write S3GW lowercase: 

```suggestion
    -> Apps -> Repositories -> Create s3gw using the s3gw-charts Git URL https://github.com/aquarist-labs/s3gw-charts and the main branch
```

File: charts/s3gw/README.md [hunk 2]
```diff
@@ -12,6 +12,17 @@ Then change into the repository and install using Helm:
...
+    -> Apps -> Charts -> Install Traefik
+    -> Apps -> Charts -> Install S3GW -> Storage -> Storage Type: pvc -> PVC Name: s3gw-pvc
```
Comment (giubacc) [inline charts/s3gw/README.md:23]: for consistency, I would rather write S3GW lowercase:

```suggestion
    -> Apps -> Charts -> Install s3gw -> Storage -> Storage Type: pvc -> PVC Name: s3gw-pvc
```

**Review by giubacc — CHANGES_REQUESTED** [review]

**Comment by giubacc** [comment]
@abonillabeeche would you mind to rebase your last commit and force push it with your commit?
We should avoid for the future proposing commits via github because they cannot be signed.

**Comment by jhmarina** [comment]
After having a chat with @abonillabeeche we agreed that I'll include the changes introduced in this PR on a different issue:   * On the charts repo, add the information added by @abonillabeeche [here](https://github.com/aquarist-labs/s3gw-charts/pull/34)

**Pull request closed without merging**